### PR TITLE
 Pipeline, Artyom Krivtsov - 22FPL2

### PR DESCRIPTION
**Pull request by artyom6k**

### CODE REVIEW

File: lab_6_pipeline/pipeline.py [hunk 1]
```diff
@@ -4,34 +4,45 @@
 # pylint: disable=too-few-public-methods, unused-import, undefined-variable, too-many-nested-blocks
 import pathlib
 
-try:
```
Comment (marina-kaz) [inline lab_6_pipeline/pipeline.py:7]: Why did you have to remove it?

**Review by marina-kaz — COMMENTED** [review]
Well, it's a start. Keep going!

File: lab_6_pipeline/pipeline.py (hunk 1 above)
Comment (artyom6k) [reply lab_6_pipeline/pipeline.py]: 🤦‍♂️

**Comment by marina-kaz** [comment]
Little changes since my last review.

**Pull request closed without merging**